projects/core/src/error-handling/http-error-handler/http-error-handler.interceptor.ts

```diff
@@ -42,4 +59,19 @@ export class HttpErrorHandlerInterceptor implements HttpInterceptor {
   protected handleError(error: HttpErrorResponse): void {
     this.errorHandler.handleError(error);
   }
+
+  /**
+   * Checks if the error corresponds to a CMS page not found.
+   *
+   * @param error - The error object to check.
+   * @returns `true` if the error corresponds to a CMS page not found, `false` otherwise.
+   */
+  protected isCmsPageNotFound(error: unknown): boolean {
```

Comment:
```suggestion
  /**
   * Checks if the error corresponds to a CMS page not found HTTP error
   *
   * @param error - The error object to check.
   * @returns `true` if the error corresponds to a CMS page not found HTTP error, `false` otherwise.
   */
  protected isCmsPageNotFoundHttpError(error: unknown): boolean {
```